src/helpers/isEqual.js

```diff
@@ -0,0 +1,288 @@
+/* eslint-disable */
+
+/**
+ * This file is taken from
+ * https://github.com/Reactive-Extensions/RxJS/blob/8f12f812d497acf639588e90f74d504a9fc801ec/src/core/internal/isequal.js
+ *
+ * ...and then slightly modified
+ */
```

Comment:
The file is crazy. And just for deep equality. I would love to move it, and we can when custom elements are out.